Nginx-Certbot.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Configure Nginx and TLS Certifications on Your Server
+layout: topic
+categories: guide
+---
+
+Now that you have a domain and a web server, you need to configure 
+the server to serve content to your domain.
+
+The two most common web server software suites are 
+[`nginx`](https://nginx.org/) and [`apache`](https://www.apache.org/). 
+This wiki page will cover web server configuration with `nginx` 
+because it is easier to configure for first-time users. The wiki page 
+assumes that the machine used to serve content is a Digital Ocean 
+droplet, but the specific steps won't differ widely between services.
+
+## Step 1: Install `nginx`
+
+Installation is relatively easy.
+- ssh into your droplet.
+- To confirm that the software is not currently installed, run the 
+  following command:<br>
```

Comment:
I'm not certain if markup plays nice with the wiki site, I would consider just using whitespace instead.

Reply:
Removed the EOL `<br>` tags for now. Addressed in 7c1a9d051f9c55fa86e76013b274a40a644cea45